src/icon-tab-bar.scss

```diff
@@ -737,6 +771,14 @@ $fd-icon-tab-bar-semantic-values: (
         }
       }
 
+      min-width: 7rem;
```

Comment:
Also wondering if we want to leave this up to the application developer

Reply:
I think you are right it is unnecessary